plugins/python-build/share/python-build/3.5.10

```diff
@@ -1,6 +1,6 @@
 prefer_openssl11
-install_package "openssl-1.1.0j" "https://www.openssl.org/source/old/1.1.0/openssl-1.1.0j.tar.gz#31bec6c203ce1a8e93d5994f4ed304c63ccf07676118b6634edded12ad1b3246" mac_openssl --if has_broken_mac_openssl
-install_package "readline-8.0" "https://ftpmirror.gnu.org/readline/readline-8.0.tar.gz#e339f51971478d369f8a053a330a190781acb9864cf4c541060f12078948e461" mac_readline --if has_broken_mac_readline
+install_package "openssl-1.1.1k" "https://www.openssl.org/source/openssl-1.1.1k.tar.gz#892a0875b9872acd04a9fde79b1f943075d5ea162415de3047c327df33fbaee5" mac_openssl --if has_broken_mac_openssl
+install_package "readline-8.1" "https://ftpmirror.gnu.org/readline/readline-8.1.tar.gz#f8ceb4ee131e3232226a17f51b164afc46cd0b9e6cef344be87c65962cb82b02" mac_readline --if has_broken_mac_readline
```

Comment:
This has no effect: these are only used in MacOS without the corresponding Homebrew packages.

Look at other patches for 3.6.15 and their commit history. We've had an issue with the specific error message that you're getting.

Reply:
Moreover, this changes version lines, so unless you're sure 3.5 supports these new version lines, better leave them alone.

Reply:
Aha. I read https://github.com/pyenv/pyenv/issues/1737#issuecomment-781647584 and I noticed that for other versions of Python, updating dependencies seemed to work, so I thought I'd give this a shot.

But thinking more about this, somehow v3.5.10 used to build correctly and find the `sendfile` function before. Updating dependencies shouldn't change that.

So, I reverted the change.